spec/models/category_list_spec.rb

```diff
@@ -116,7 +116,8 @@
   end
 
   describe 'category order' do
-    let(:category_ids) { CategoryList.new(Guardian.new(admin)).categories.map(&:id) - [SiteSetting.uncategorized_category_id] }
```

Comment:
This might be one of those few cases were adding a helper to a test could simplify things a lot. These lines are very similar and could use a function.